torch/_ops.py

```diff
@@ -25,6 +25,109 @@ def dl_open_guard():
     if _SET_GLOBAL_FLAGS:
         sys.setdlopenflags(old_flags)
 
+# Each OpOverload object contains pointer to a a specific operator overload, a pointer to the parent `OpOverloadPacket` object.
+# You can obtain an OpOverload object through attribute query on OpOverloadPacket.
+class OpOverload:
+    def __init__(self, overloadpacket, op, schema):
+        self._op = op
+        self._schema = schema
+        self._overloadpacket = overloadpacket
+
+    # it's a no-op since OpOverload object is immutable and must be for a given op overload.
```

Comment:
missing "unique"?
(same for the one below)

Reply:
I'll add it as a TODO for the follow-up PR. don't really wanna deal with exporting the internal changes:)